### DISCUSSION TITLE
Workflow Activity Bar - How to classify inputs?

### DISCUSSION THREAD

**Discussion by jmchilton** [galaxyproject/galaxy#19214] [General Discussion]
To me it is weird that inputs are classified as tools but workflows are something categorically different from the editor bar prespective. Workflows behave in some ways a lot more like tools than inputs do like either right?

<img width="376" alt="Screenshot 2024-11-27 at 11 21 03 AM" src="https://github.com/user-attachments/assets/d9bffb0d-7a90-42bd-b191-7c7db39d21e5">

I would suggest we create a new top level thing for inputs or we relabel "Tools" in the bar to "Tools and Inputs".

**Comment by mvdbeek** [comment]
👍 to adding an inputs activity
